### DISCUSSION TITLE
Tabs: How to SSR all tabs content instead of just first tab?

### DISCUSSION THREAD

**Discussion by IlirEdis** [radix-ui/primitives#2711] [Help]
Hi there. 
I'm using Tabs component from Shadcn which is a Radix primitive so thats why i thought this is the best place to ask.

I have some Tabs in which i have some links inside each of these tabs content so i'd like them to be rendered in the source code initially for SEO purposes. For now, in the source code i can only see the content from the first tab. How can i render all of the tabs content initially?

**Comment by benoitgrelard** [comment]
This isn't currently possible with `Tabs`.
If you require a tab-like interface (visually) but need SEO you're probably rather looking at a different pattern which `NavigationMenu` could cover well (ie. actual links to actual pages/nested routes.)

**Reply by IlirEdis** [reply]
Sorry for the confusion! 
I don't want to navigate to other links when switching the tabs but i have some links in the tab content which i want to be "seen" by the search engines. However this is not important. I wanted to say that simply i want the tab content to be "seen".

Any other suggestion with radix or i should go custom in this case?

Thanks!

**Reply by benoitgrelard** [reply]
You mean to have the content of all the tabs mounted even though they're hidden?

**Reply by IlirEdis** [reply]
Yes.

**Reply by benoitgrelard** [reply]
This isn't something we support right now, but we have discussed the possibility of adding some mount options. Have a read through this discussion: https://github.com/radix-ui/primitives/discussions/855.

I will close yours as it's a duplicate of that one essentially.

**Reply by IlirEdis** [reply]
Ok. Thanks!

**Reply by psupplyceo** [reply]
Can you render the navigationMenu serverside? I cant seem to get it to work

**Comment by benoitgrelard** [comment]
Duplicate of #855
